src/main/java/hudson/plugins/locale/PluginImpl.java

```diff
@@ -20,7 +24,8 @@
 /**
  * @author Kohsuke Kawaguchi
  */
-public class PluginImpl extends Plugin {
+@Extension

```

Comment:
Add `@Symbol`?

Reply:
require to upgrade core to 2.2+, is it ok ?

Reply:
Yes. I would just go straight to Java 8 and 2.60.3